q2_demux/_format.py

```diff
@@ -18,6 +18,17 @@ class EMPMultiplexedDirFmt(model.DirectoryFormat):
         r'barcodes.fastq.gz', format=FastqGzFormat)
 
 
+class PairedEMPMultiplexedDirFmt(model.DirectoryFormat):
```

Comment:
How's this:

EMP implies multiplexed reads (if they're already demultiplexed, we don't care that they were once EMP reads), so:

`EMPMultiplexedDirFmt` -> `EMPSingleEndDirFmt`
`PairedEMPMultiplexedDirFmt` -> `EMPPairedEndDirFmt`

and then:

`EMPMultiplexedSingleEndDirFmt` -> `EMPSingleEndCasavaDirFmt`
`EMPMultiplexedPairedEndDirFmt` -> `EMPPairedEndCasavaDirFmt`

because the file names in that dir format are generated by Casava.